opensrp-app/src/main/java/org/smartregister/location/helper/LocationHelper.java

```diff
@@ -168,7 +168,7 @@ public String getOpenMrsLocationId(@NonNull String locationName) {
         } catch (Exception e) {
             Timber.e(e);
         }
-        return locationName.equals(response) ? AllConstants.ADVANCED_DATA_CAPTURE_STRATEGY_PREFIX + locationName.trim().replaceAll(" ", "-").toLowerCase(Locale.ENGLISH) : response;
+        return response;
```

Comment:
Not sure I understand this change?

Reply:
@ndegwamartin  made this change an just reverting it since it breaks our code. I have discussed with him

Reply:
cool